cabalscan/src/CabalScan/Rules.hs

```diff
@@ -16,27 +17,33 @@ import Data.Text (Text)
 -- >   , deps = ["protolude"]
 -- >   , extraLibraries = ["libsodium"]
 -- >   , tools = [ToolName "tasty-discover" "tasty-discover"]
--- >   , attrs = [ ("srcs", ["src/Main.hs"])
--- >             , ("ghcopts", "-DFOO=1")
--- >             ]
+-- >   , version = "0.1.0.0"]
+-- >   , hidden_modules = Just ("PackageA.Other.B" :| ["PackageA.Other.D"])
+-- >   , dataAttr = Nothing
+-- >   , srcs = ["src/Main.hs"]
```

Comment:
```suggestion
-- >   , ghcopts = ["-DFOO=1"]
```

Reply:
hm, ghcopts go elsewhere. This example is not up to date. We can either point that fields are missing or try to keep it up to date.

Reply:
How do you want to bring it up to date? It would require also mentioning `ImportData`.